packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -52,6 +52,10 @@ abstract class RunCommandBase extends FlutterCommand with DeviceBasedDevelopment
             'By default, this is not enabled to reduce the overhead. '
             'This is only available in profile or debug build. ',
       )
+      ..addFlag('purge-persistent-cache',
+        negatable: false,
+        help: 'Remove all existing persistent cache.',
```

Comment:
please add more context about what this flag does, and why it would need to be passed